src/vm/vm.ts

```diff
@@ -119,21 +125,25 @@ function step(n: number = 0): void {
       ? dstAddr
       : runSheet.getRange(dstAddr).getValue();
 
-  // Set formula for current opcode: dst and res
+  // Set formula for current opcode, op0, op1 and dst
+  runSheet.getRange(`${op0Column}${n + 2}`).setFormula(`=${op0Addr}`);
+  runSheet.getRange(`${op1Column}${n + 2}`).setFormula(`=${op1Addr}`);
   runSheet.getRange(`${dstColumn}${n + 2}`).setFormula(`=${dstAddr}`);
   switch (instruction.ResLogic) {
     case ResLogics.Add:
       runSheet
         .getRange(`${resColumn}${n + 2}`)
-        .setFormula(`=${op0Addr} + ${op1Addr}`);
+        .setFormula(`=${op0Column}${n + 2} + ${op1Column}${n + 2}`);
       break;
     case ResLogics.Mul:
       runSheet
         .getRange(`${resColumn}${n + 2}`)
-        .setFormula(`=${op0Addr} * ${op1Addr}`);
+        .setFormula(`=${op0Column}${n + 2} * ${op1Column}${n + 2}`);
       break;
     case ResLogics.Op1:
-      runSheet.getRange(`${resColumn}${n + 2}`).setFormula(`=${op1Addr}`);
+      runSheet
+        .getRange(`${resColumn}${n + 2}`)
+        .setFormula(`=${op1Column}${n + 2}`);
       break;
```

Comment:
why did you have to reintroduce op0 and op1 column?

Reply:
There is no reason indeed, my bad